src/Models/Lotto.js

```diff
@@ -0,0 +1,34 @@
+import { OutputComment } from '../Constants/display.js';
+import Errors from '../Constants/Errors.js';
+import throwError from '../Utils/throwError.js';
+import Rules from '../Utils/Rules.js';
+
+class Lotto {
+  #numbers;
+
+  constructor(numbers) {
+    this.#validate(numbers);
+    this.#numbers = numbers.sort((a, b) => a - b);
+  }
+
+  #validate(numbers) {
+    if (Rules.isWrongLength(numbers, 6))
+      throwError(Errors.Lotto.IS_WRONG_LENGTH);
+    if (Rules.isNotNumberInNumbers(numbers))
+      throwError(Errors.Lotto.NOT_NUMBER_VALUE);
+    if (Rules.isDuplicatedValue(numbers))
+      throwError(Errors.Lotto.IS_DUPLICATE_VALUE);
+    if (Rules.isNotRangedValueInNumbers(numbers))
+      throwError(Errors.Lotto.NOT_RANGED_VALUE);
+  }
+
+  getNumbers() {
+    return this.#numbers;
+  }
+
+  getPrintString() {
+    return `[${this.#numbers.join(OutputComment.SEPERATOR)}]`;
+  }
```

Comment:
오호 이런 방법도 되는군요 좋은 것 같네요👍👍👍

Reply:
```suggestion
  formatLottoString() {
    return `[${this.#numbers.join(OutputComment.SEPERATOR)}]`;
  }
```
로또 번호를 로또 문자열에 맞게 포맷 형식을 바꾸는 것 같아서 `formatLottoString`같은 네이밍은 어떨까요?

Reply:
오 너무 좋습니다! 이번 미션에서 format이라는 키워드를 적극적으로 사용해봐야겠어요 👍